types/responses.ts

```diff
@@ -124,6 +159,10 @@ export interface GenerateContentCandidate {
   finishMessage?: string;
   safetyRatings?: SafetyRating[];
   citationMetadata?: CitationMetadata;
+  /** Average log probability score of the candidate. */
+  avgLogProbs?: number;
+  /** Log-likelihood scores for the response tokens and top tokens. */
+  logProbsResult?: LogprobsResult;
```

Comment:
Since the proto has `logprobs_result`, this would translate to `logprobsResult` in camelcase. This should be the capitalization in the response, you can double check with curl I think.